src/zero/boot.clj

```diff
@@ -64,18 +64,23 @@
 
 (defn clone-repos
   "Return a map of zero/the-github-repos clones in a :tmp directory.
-   Delete the :tmp directory at some point."
+   Delete the :tmp directory at some point.
+
+   Specifically omits [[zero/the-name]]'s repo since it isn't needed
+   for the version."
   []
-  (let [tmp (str "CLONE_" (UUID/randomUUID))]
+  (let [tmp (str "CLONE_" (UUID/randomUUID))
+        the-github-repos-no-wfl (dissoc zero/the-github-repos zero/the-name)
```

Comment:
are `the-github-repos` and `the-other-github-repos` used anywhere else? Maybe you can just remove wfl from them?

Reply:
Yeah wfl is specifically pulled out of `the-github-repos` elsewhere in `boot.clj`. `the-other-github-repos` isn't used anywhere else but it has the exact same repos as `the-github-repos` with a different URL scheme used in case of a failure with the first. I suppose this means we're keeping a backup URL for wfl that we wouldn't ever use, but there's a part of me that thinks keeping the parallelism between the two lists is more helpful than molding the lists to the behavior of this function.

Reply:
Update: did more research and the backup URLs don't work and aren't supposed to as far as I can tell from GitHub's documentation. Given that I think they'd only help in the case of weird DNS issues on GitHub's raw domain (?) I removed the entire list of backup URLs. I also removed the try/catch around it in the versioning function so in case GitHub does go completely down we won't swallow the errors (same behavior as before).

Reply:
Okay, last update. The backup URLs *are* used, exclusively in GitHub actions it seems, for accessing the private repos without other overhead. I documented that and re-organized the data structure to better represent it.